### PR TITLE
Adaptate terminal to the upstream

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
none.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16212

Fix terminal compilation after upstream changes: https://github.com/eclipse-theia/theia/pull/7121

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by vzhukovskii** [comment]
Linked issue in che-theia: https://github.com/eclipse/che/issues/16212

**Comment by vzhukovskii** [comment]
Let's wait until https://github.com/eclipse-theia/theia/pull/7121 will be merged

**Comment by AndrienkoAleksandr** [comment]
> terminal extension has dependency: "xterm": "3.9.2",

Thanks for comment. I think current yarn which we are using skip this version and use upstream xterm... So we can remove this version from package.json. It is transitive dependency for us. When theia ci complete work I will update yarn.lock.

**Comment by RomanNikitenko** [comment]
should we replace [import](https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-terminal/src/browser/terminal-frontend-module.ts#L19) by `'xterm/css/xterm.css'`?
wdyt?

**Comment by AndrienkoAleksandr** [comment]
> 'xterm/css/xterm.css'

Nice catch. I tested - styles correctly applied and they are working. So I think it works from upstream code and I guess we can skip injection this styles, but I will take a look.

**Comment by RomanNikitenko** [comment]
> So we can remove this version from package.json.

> So I think it works from upstream code and I guess we can skip injection this styles, but I will take a look.

Please remove `import` and `dependency` within the current PR if they are not required for `che-theia`
thank you!

File: generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json [hunk 1]
```diff
@@ -175,6 +175,24 @@
...
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
```
Comment (RomanNikitenko) [inline generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json]: these changes are the result of https://github.com/eclipse/che/issues/15916
it's very annoying bug...

**Comment by azatsarynnyy** [comment]
crw-ci-test

File: generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json (hunk 1 above)
Comment (AndrienkoAleksandr) [reply generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json]: OK then -> https://github.com/eclipse/che-theia/pull/655

**Comment by azatsarynnyy** [comment]
[ci-build]

**Pull request merged**